packages/client/src/services/useAuth/index.ts

```diff
@@ -0,0 +1,21 @@
+import { useEffect, useState } from 'react'
```

Comment:
Might it better to rename hook to useGetLoggedinUser or somehow, because we dont make authentification here. And I think you need not just create hook, but use it also in this task

Reply:
Переименовал. Вроде бы для применения этого хука отдельная задачка